### ISSUE TITLE
The wramup-ratio of QQP in RoBERTa is 0.25, not 0.06

### ISSUE THREAD

**Issue by guolinke** [pytorch/fairseq#2482]
#### What is your question?

Why QQP use the much larger warm-up ratio than other GLUE tasks?
I searched the paper of RoBERTa, and cannot find any explanation.

#### Code

   

https://github.com/pytorch/fairseq/blob/master/examples/roberta/README.glue.md#3-fine-tuning-on-glue-task

**Comment by guolinke** [comment]
@ngoyal2707 can you help with this question? thanks very much.

**Comment by sashka732** [comment]
Can someone help with this question? I am having trouble reproducing the results from the paper, and I am wondering if this is the cause.
